Update troubleshoot-bad-request.md to add new error (#122365)
Azure/azure-content: articles/cosmos-db/nosql/troubleshoot-bad-request.md

### Solution
Send the partition key value parameter that matches the document property value.

> ## <a name="partition-key-value-precision-loss"></a>Numeric partition key value precision loss

Reviewer: Remove anchor, the title itself already is an anchor. Other titles have it because their names changed over time

Reviewer: ```suggestion
## Numeric partition key value precision loss
```

On this scenario, it's common to see errors like:

*The requested partition key is out of key range, possibly because of loss of precision of partition key value*

> A response with this error is likely to be caused by an operation on a document with a numeric partition key whose value is outside what is supported by Azure Cosmos DB. Refer [Per-item limits](https://learn.microsoft.com/en-us/azure/cosmos-db/concepts-limits#per-item-limits) for the maximum length of numeric property value.

Reviewer 2: ```suggestion
A response with this error is likely to be caused by an operation on a document with a numeric partition key whose value is outside what is supported by Azure Cosmos DB. Refer [Per-item limits](/azure/cosmos-db/concepts-limits#per-item-limits) for the maximum length of numeric property value.
```

Reviewer: Another alternative is 
```suggestion
A response with this error is likely to be caused by an operation on a document with a numeric partition key whose value is outside what is supported by Azure Cosmos DB. Refer [Per-item limits](../concepts-limits.md#per-item-limits) for the maximum length of numeric property value.
```

> On this scenario, it's common to see errors like:
> 
> *The requested partition key is out of key range, possibly because of loss of precision of partition key value*
> 
> A response with this error is likely to be caused by an operation on a document with a numeric partition key whose value is outside what is supported by Azure Cosmos DB. Refer [Per-item limits](/azure/cosmos-db/concepts-limits#per-item-limits) for the maximum length of numeric property value.
> 
> ### Solution
> Consider using type `string` for partition key if requiring precise numeric values.

Reviewer 2: ```suggestion
On this scenario, it's common to see errors like:

*The requested partition key is out of key range, possibly because of loss of precision of partition key value*

A response with this error is likely to be caused by an operation on a document with a numeric partition key whose value is outside what is supported by Azure Cosmos DB. See [Per-item limits](/azure/cosmos-db/concepts-limits#per-item-limits) for the maximum length of numeric property value.

### Solution
Consider using type `string` for partition key if requiring precise numeric values.
```

> Consider using type `string` for partition key.

Reviewer: nit: Maybe a bit more specific?

```suggestion
Consider using type `string` for partition key if requiring precise numeric values.
```
